engine/src/main/java/org/terasology/engine/GameEngine.java

```diff
@@ -45,19 +62,14 @@
     void close();
 
     /**
-     * @return Whether the engine has been uninitialized
+     * @return Whether the engine is running - this is true from the point run() is called to the point shutdown is complete
      */
-    boolean isUninitialized();
-
-    /**
-     * @return Whether the engine has been initialized
-     */
-    boolean isInitialized();
```

Comment:
Correct. I also don't think the idea of initialization carrying on between shut down and disposal needs to be exposed either.